src/main.rs

```diff
@@ -30,7 +31,7 @@ async fn main() {
     setup_backtrace();
 
     let opt = Options::parse();
-    let vm = RollupVM::new(1.into());
+    let vm = RollupVM::new(NamespaceId::from(1_u64));
```

Comment:
we're likely doing the demo on L1 but would this change in the future? e.g. when the light client is deployed on l1. If so, should this come from an env variable?

Reply:
This is an example rollup which has a random chainId